pkg/services/authz/zanzana/client/client.go

```diff
@@ -115,8 +112,34 @@ func (c *Client) Compile(ctx context.Context, id claims.AuthInfo, req authz.List
 		return nil, err
 	}
 
-	// FIXME: implement checker
-	return func(namespace, name, folder string) bool { return false }, nil
+	return newItemChecker(res), nil
+}
+
+func newItemChecker(res *authzextv1.ListResponse) authz.ItemChecker {
+	// if we can see all resource of this type we can just return a function that always return true
+	if res.GetAll() {
```

Comment:
is this a heavy operation? 🤔

Reply:
What do you mean?

Reply:
It's just getting a field in the response, it gets `All` field value which is bool.